src/App.tsx

```diff
@@ -3,25 +3,21 @@ import logo from './logo.svg';
 import './App.css';
 import NASA from "./components/NASA";
 import TitleParagraphImageComponent from "./components/TitleParagraphImageComponent";
+import ButtonWithCounter from "./components/ButtonWithCounter";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-          <TitleParagraphImageComponent title={"Just a cat"} p1={"paragraph1"} p2={"paragraph2"} image={"http://placekitten.com/200/300"}></TitleParagraphImageComponent>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      <NASA></NASA>
-      </header>
-    </div>
-  );
+    return (
+        <div className="App">
+            <header className="App-header">
+                <img src={logo} className="App-logo" alt="logo"/>
+                <ButtonWithCounter></ButtonWithCounter>
```

Comment:
We can just do 
```suggestion
                <ButtonWithCounter/>
```
when we're not passing anything through to a component